godel_robots/abb/godel_irb2400/godel_irb2400_support/config/scan_plan.yaml

```diff
@@ -0,0 +1,10 @@
+scan_plan:
+  scan_width: 0.01
+  traverse_spd: 0.0025
+  approach_distance: 0.15
+  overlap: 0.001
+  margin: 0
+  quality_metric: "rms"
```

Comment:
This will need to be fixed as we go forward without string based quality-metrics